modules/platform/nuxeo-drive-server/nuxeo-drive-core/src/test/java/org/nuxeo/drive/fixtures/GroupChangesTestSuite.java

```diff
@@ -345,4 +345,12 @@ protected void testGroupChangesWithAncestorGroups(String ancestorGroup) {
         testGroupChanges(syncRoot, "defaultSyncRootFolderItemFactory", GROUP_1, true);
     }
 
+    @Override
+    protected void commitAndWaitForAsyncCompletion() {
+        // wait for the NuxeoDriveGroupUpdateListener asynchronous listener
+        super.commitAndWaitForAsyncCompletion();
+        // wait again for the FireGroupUpdatedEventAction bulk action, submitted by the NuxeoDriveGroupUpdateListener
```

Comment:
to be exact we are not waiting for the bulk action but for async works triggered by the bulk action

Reply:
Yes! Let me update this.